src/planscape/plan/views.py

```diff
@@ -122,12 +121,15 @@ def delete(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Error in delete: " + str(e))
 
 
-def get_plan_by_id(params: QueryDict):
+def get_plan_by_id(user, params: QueryDict):
```

Comment:
This seems to have caused an error with the `plan/scores` endpoint (I'm guessing because that endpoint doesn't have unit test coverage yet). @strangeties could you update the code for that endpoint to add the new argument to `get_plan_by_id`?

![image](https://user-images.githubusercontent.com/10444733/214162629-9c3f735b-f64a-4a49-b8b3-b67380f298d7.png)

Reply:
Followup: when I update the backend code myself to use `None` as the user param for `get_plan_by_id`, I get a new error:

![image](https://user-images.githubusercontent.com/10444733/214163678-590f09dd-d36e-41ba-b0d1-217a5ecda6e7.png)

I'm not able to tell what's causing this one :(

Reply:
sure - I'll prioritize updating this.

Reply:
@thequeenofspades which plan id are you requesting and what are you expecting? This also reminded me of another bug that  I just sent out a fix +test for #403